lms/static/sass/discussion/elements/_navigation.scss

```diff
@@ -14,15 +14,15 @@
 // Header
 // ------
 .forum-nav-header {
-  @include box-sizing(border-box);
+  box-sizing: border-box;
   display: table;
-  border-bottom: 1px solid $gray-l2;
+  border-bottom: 1px solid $forum-color-border;
   width: 100%;
   background-color: $gray-l3;
 }
 
 .forum-nav-browse {
-    @include box-sizing(border-box);
+    box-sizing: border-box;
```

Comment:
I agree with both points. There is a *lot* of duplication in these stylesheets that I'd like to clean up, but I won't do it in this PR.